.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: go
 go:
-  - '1.11.x'
-sudo: required
-services:
-  - docker
```

Comment:
Isn't that required? Or do we always get a docker daemon?

Reply:
sudo and docker are both provided by default, so there's no need to specify them.